clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -157,14 +193,27 @@ object UncommittedGarbageCollector {
       firstSlice: String,
       startTime: java.time.Instant,
       success: Boolean,
+      expiredAddresses: DataFrame,
       removed: DataFrame
```

Comment:
What's the difference between these 2? Aren't they mutually exclusive?

Reply:
The expired address is the addresses we marked, and the removed ones are the addresses we sweep. They can be identical if no other mark id was given, or empty if we did only one operation.